### DISCUSSION TITLE
Smooth transition between two scrolls areas

### DISCUSSION THREAD

**Discussion by chelopezlo** [idiotWu/smooth-scrollbar#474] [Q&A]
Hello!!

I want to do a page like https://www.sendpotion.com/, where there's a section that has a horizontal scroll and continues scrolling vertically after it reaches the end of the section.
I have made the horizontal scroll plugin like in this entry https://github.com/idiotWu/smooth-scrollbar/discussions/359 but I can't get the transition between vertical and horizontal scrolling to change smoothly. It continues scrolling in both directions until the vertical scroll stops completely. If I move the mouse while scrolling it works (the vertical scroll stops).

Can someone help me pls?! I've trying to do this for a few days with no luck.

Thanks!!

**Comment by dolphin-wood** [comment]
It seems that they are using [GASP](https://greensock.com/) in https://www.sendpotion.com. Not familiar with GASP, but I found some helpful resources: 

- [scrollTrigger with smooth-scrollbar](https://greensock.com/forums/topic/24610-scrolltrigger-with-smooth-scrollbar/)
- [ScrollTrigger.scrollerProxy() smooth-scrollbar demo](https://codepen.io/GreenSock/pen/GRovmPr)
